Possible issue with open method when using rocksdict

## Steps to reproduce

Trying to run with store=rocksdb:// on windows

## Expected behavior

I expect it can store data in tables and recover quickly when changelogs are long

## Actual behavior

Errors with [^----Store: rocksdb:<table name>]: DB for partition 2 retries timed out.

I tracked it down to the fact that the resource module (used in faust/utils/platforms) is a Unix only thing so the `_max_open_files` variable defaults to `None`, which then flows on to a crash because it's trying to set the `_max_open_files` on `db.options` (in faust/stores/rocksdb.py) which does not like a None.
Setting this default to 1 instead of None resolved that issue, but then I encountered another issue where the open method tries to call `rocksdict.AccessType.ReadWrite`, which doesn't exist.
Looking at the `AccessType` method shows it doesn't have `ReadWrite`, but does have `read_write`, so I replaced with `rocksdict.AccessType.read_write()` and it worked.

I'm fairly certain there is no issue here except that I'm trying to use this with Windows. But I did get it working, so I'm not sure whether there's something here to explore or this is a big don't.

So you're seeing this then?
```python
[INFO] Executing _on_partitions_assigned
[INFO] opening partition 2 for gen id 3 app id 3 #<== Numbers here might be different each time
[WARNING] [^----Store: rocksdb:<table_name>]: DB for partition 2 retries timed out
[INFO] Closing rocksdb on stop
[ERROR] [^-App]: Crashed reason=TypeError("argument 'nfiles': 'NoneType' object cannot be interpreted as an integer")
```
That would be interesting, as it would imply the issue for you is coming from the platforms module. I'd suggest adding some debug statements in there (just for yourself) so you can see why it's returning the None. If it's the `ImportError` then ensure you have the resource package installed (although I think that's default for Linux 🤷). If it's returning None in the next section then that's strange and requires someone who properly understands this to assist - I'm no pro with this stuff but can dabble enough to duct tape a temporary solution.

--- a/faust/stores/rocksdb.py
+++ b/faust/stores/rocksdb.py
@@ -143,7 +143,7 @@ class RocksDBOptions:
                 [rocksdict.DBPath(str(path), self.target_file_size_base)]
             )
             db_access_type = (
-                rocksdict.AccessType.ReadWrite
+                rocksdict.AccessType.read_write()
                 if self.ttl is None
                 else rocksdict.AccessType.with_ttl(self.ttl)
             )

